### ISSUE TITLE
atom-vue prevents use of hyphen as word break on .vue files?

### ISSUE THREAD

**Issue by NoahCardoza** [hedefalk/atom-vue#83]
Yesterday I noticed when option-arrow keying around, that the hyphen ("-") was not being treated as a point to break words.  After a day of this, it became very frustrating as you can imagine.

I checked my default settings, they included a hyphen in my list of word breaking characters.
After disabling and re-enabling all my packages I have narrowed it down to this one.

I would fix it if I knew how, but alas I don't... If it wouldn't be too much to ask it would be nice to have this fixed.

Thanks

**Comment by aidistan** [comment]
Hi @NoahCardoza, sorry for no response for quite a long time. Maybe you have not used this packege any more. But if you are still struggling with this frustrating issue, could you please provide an example (like examples in [tests](https://github.com/hedefalk/atom-vue/blob/master/test/template.vue)) and a few screenshots? It would be nice for us to fix the issue with it.

**Comment by NoahCardoza** [comment]
I've been pretty busy with school lately and haven't worked on my Vue side projects lately.
If I get a chance I'll see what I can do about a more detailed bug report. Who knows it might be fixed by now.

**Comment by NoahCardoza** [comment]
Well, I figured it wouldn't be too hard to check so I did it anyway.
I am happy to report the bug/whatever-it-was seems to be fixed. 

To explain the problem a little better though,
If my cursor was `|` and at this position: `<|my-component>` and I option-left-arrowed the cursor would end up `<my-component|>` instead of `<my|-component>` which is was doing before the package was installed. But like I said this doesn't seem to nbe an issue anymore.

Thanks!
